examples/plot_clip.py

```diff
@@ -0,0 +1,125 @@
+"""
+Clip Vector Data with EarthPy
+==================================================================
+
+Learn how to clip point, line, or polygon geometries with a
```

Comment:
Super minor - change "with" to "to", as it informs the user that the clip function "clips to something".